scripts/examples/generate_book_cover.py

```
@@ -0,0 +1,144 @@
+import urllib.request
+import urllib.parse
+
+server_address = "localhost:8188"
+client_id = str(uuid.uuid4())
+
+def queue_prompt(prompt):
+    p = {"prompt": prompt, "client_id": client_id}
+    data = json.dumps(p).encode('utf-8')
+    try:
+        req = urllib.request.Request(f"http://{server_address}/prompt", data=data)
+        return json.loads(urllib.request.urlopen(req).read())
```

Ruff (0.15.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)